tests/end_to_end/conftest.py

```diff
@@ -108,6 +108,26 @@ def others(addresses):
     return addresses[2:]
 
 
+@pytest_asyncio.fixture(scope="function")
```

Comment:
made this function scope as I want to have a new account at disposition for each call